packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -84,6 +84,15 @@ export type ChatCompositeOptions = {
    * @beta
    */
   fileSharing?: FileSharingOptions;
+
+  /* @conditional-compile-remove(rich-text-editor) */
+  /**
+   * Show or hide the Rich Text Editor feature.
+   * @defaultValue false
+   *
+   * @beta
+   */
+  richTextEditor?: boolean;
```

Comment:
We probably need to have an object here instead of boolean so we won't need to change it later. But it's not very clear what should be inside of it.